src/UserInterface/Image/createLabelMapColorWidget.js

```diff
@@ -101,16 +113,23 @@ function createLabelMapColorWidget(store, uiContainer) {
   applyContrastSensitiveStyle(store, 'invertibleButton', sliderEntryDiv)
   const volume = store.imageUI.representationProxy.getVolumes()[0]
   const volumeProperty = volume.getProperty()
+  const sliceActor = store.imageUI.representationProxy.getActors()[0]
+  const sliceProperty = sliceActor.getProperty()
   function updateLabelMapColorOpacity() {
     const labelMapOpacity = store.imageUI.labelMapOpacity
     opacityElement.value = labelMapOpacity
     const numberOfComponents = store.imageUI.numberOfComponents
+    for (let c = 0; c < numberOfComponents; ++c) {
```

Comment:
Nitpick: In JavaScript `c++` is preferred for performance, the opposite of C++ :-(